.github/workflows/release-nym-vpn-cli.yml

```diff
@@ -128,10 +133,14 @@ jobs:
         with:
           components: rustfmt, clippy
 
-      - name: Install extra arch macos
-        if: env.PLATFORM_ARCH == 'macos_universal'
+      - name: Install extra arch apple
+        if: contains(matrix.os, 'custom-runner-mac-m1') || contains(matrix.os, 'macos-14')
         run: |
           rustup target add x86_64-apple-darwin
+          rustup target add aarch64-apple-ios
+          rustup target add x86_64-apple-ios
```

Comment:
Don't need this one. Everything is 64bit.

Reply:
Those are just the names of the targets in rust. Yep, no 32 bit in here :D